packages/validator/src/cli-validator/utils/cli-options.js

```diff
@@ -49,15 +70,23 @@ function createCLIOptions() {
       'display only the summary section and skip individual errors/warnings'
     )
     .option('-v, --verbose', 'display verbose results')
+    .option(
+      '-w, --warnings-limit <number>',
+      'set warnings limit to <number>',
+      parseWarningsLimit
+    )
     .version(getVersionString(), '--version')
     .showHelpAfterError()
-    .addHelpText('beforeAll', getCopyrightString())
-    .addOption(
-      new Option(
-        '--limits <filename',
-        'the name of the file containing the limits configuration'
-      ).hideHelp()
-    );
+    .addHelpText('beforeAll', getCopyrightString());
+
+  // Prevent Commander from calling process.exit() for an error.
+  command.exitOverride();
+
+  // Customize message output behavior of Commander.
+  command.configureOutput({
+    writeOut: s => console.log(s),
+    writeErr: s => console.error(s)
+  });
 
   return command;
```

Comment:
That's because I added the call to `command.exitOverride()` to prevent Commander from calling process.exit() when we detect an unknown CLI option :joy:.  I did that so that we could more easily test those error conditions.  I didn't realize that it also applied to the "help" path.  I'll need to back that out I guess.

Reply:
Actually, I think I can just catch that exception in run-validator and absorb it.  That would probably be the best of both worlds.

Reply:
I've changed runValidator slightly so that it catches and reacts to any exception that it might get from processArgs(), which would likely be an exception thrown from within the Commander code (unknown option, help was displayed, etc.).
We're now seeing these CommanderErrors being thrown for those situations because we were previously just allowing Commander to call process.exit(), but I "fixed" that (:joy:) making sure we call command.exitCallback() so that an exception is thrown before Commander gets to the process.exit() call.   Anyway, with the updated code, I'm handling any exception that we might have caught from Commander right near the call to processArgs().